OneSignalSDK/onesignal/src/main/java/com/onesignal/PushRegistratorFCM.java

```diff
@@ -43,6 +44,10 @@
 
 class PushRegistratorFCM extends PushRegistratorAbstractGoogle {
 
+   private static final String FCM_DEFAULT_PROJECT_ID = "onesignal-shared-public"; // project_info.project_id
+   private static final String FCM_DEFAULT_APP_ID = "1:754795614042:android:c682b8144a8dd52bc1ad63"; // client.client_info.mobilesdk_app_id
+   private static final String FCM_DEFAULT_API_KEY = "AIzaSyAnTLn5-_4Mc2a2P-dKUeE-aBtgyCrjlYU"; // client.api_key.current_key
+
```

Comment:
These are valid. I checked against Firebase servers and Google Cloud frontend.

Reply:
Are these hardcoded constant values of OneSignal default keys in Firebase panel?